test/functional/tests/incremental_load/test_core_pool.py

```diff
@@ -48,10 +48,11 @@ def test_attach_core_pool():
         cache.stop()
 
     with TestRun.step("Add previously used core device to core pool using --try-add flag."):
-        first_core = casadm.try_add(core_dev, cache.cache_id)
+        first_core = casadm.try_add(core_device=core_dev, cache_id=cache.cache_id, core_id=1)
```

Comment:
Instead of magic numbers "1", and "2", could you use "first_core.core_id" and "second_core.core_id"if available please? Or define for example "core_1_id =1" and "core_2_id =2" and use them here and later instead of numbers

Reply:
I defined additional variables for core and cache id.

Reply:
thank you  🤩🌼